### PR TITLE
CORE-11449 Merge Network team feature branch

### PR DESCRIPTION
**Pull request by YashNabar**
Merge temporary feature branch of network team back into main branch after beta 2 branch is cut.

### CODE REVIEW

File: tools/plugins/build.gradle [hunk 1]
```diff
@@ -134,7 +134,7 @@ tasks.register('publishOSGiImage', DeployableContainerBuilder) {
     if (project.hasProperty('cliBaseTag')) {
         baseImageTag = cliBaseTag
     } else {
-        it.baseImageTag = (cordaCliIncluded) ? "latest-local" : "unstable"
+        it.baseImageTag = (cordaCliIncluded) ? "latest-local" : "unstable-network" // not for merging back to release/os/5.0
```
Comment (dimosr) [inline tools/plugins/build.gradle]: this needs to be reverted - see inline comment

File: buildSrc/src/main/groovy/DeployableContainerBuilder.groovy [hunk 2]
```diff
@@ -296,7 +296,7 @@ abstract class DeployableContainerBuilder extends DefaultTask {
             tagContainer(builder, "${tagPrefix}${version}")
         } else if (releaseType == 'BETA' && !nightlyBuild.get()) {
             targetRepo = "corda-os-docker-unstable.software.r3.com/corda-os-${containerName}"
-            tagContainer(builder, "${tagPrefix}unstable")
+            tagContainer(builder, "${tagPrefix}unstable-network") // not for merging back to release/os/5.0
```
Comment (dimosr) [inline buildSrc/src/main/groovy/DeployableContainerBuilder.groovy]: this needs to be reverted - see inline comment

**Review by dimosr — COMMENTED** [review]
Can you also add to the description if there were any conflicts or not? If there were any conflicts, it's also good to include a git patch file with the diff, so that reviewers can review just the conflicts given the other PRs have already been reviewed properly before being merged into the feature branch.

**Review by dimosr — DISMISSED** [review]

**Review by ronanbrowne — APPROVED** [review]
lgtm (build)

**Review by dimosr — APPROVED** [review]

**Pull request merged**